pkg/controller/nodeconfigpod/sync.go

```diff
@@ -39,6 +39,11 @@ func (ncpc *Controller) sync(ctx context.Context, key string) error {
 		return fmt.Errorf("can't list pods: %w", err)
 	}
 
+	_, hasScyllaLabel := pod.Labels[naming.ClusterNameLabel]
```

Comment:
can labels be nil? you've also made a function handling that int this PR

Reply:
good point, I missed that. Moved func to `naming` and used it here as well.